src/main/kotlin/com/sourcegraph/cody/chat/ui/LLMDropdown.kt

```diff
@@ -95,4 +99,24 @@ class LLMDropdown(val project: Project, private val modelFromState: ChatModel?)
       toolTipText = CodyBundle.getString("LLMDropdown.disabled.text")
     }
   }
+
+  private fun setLlmForAgentSession(modelName: String) {
+    val sessionIdGetNow = sessionId.getNow(null) ?: return
```

Comment:
I assume it's possible to change model only after `sessionId` was already obtained, until then sector is disabled, right?
Otherwise it could lead to a problems.

Reply:
yes, that's inconvenient but correct